synapse/storage/registration.py

```diff
@@ -92,6 +92,13 @@ def get_user_by_id(self, user_id):
             query, user_id
         )
 
+    def get_all_users(self):
+        query = ("SELECT users.name FROM users")
+        return self._execute(
+            self.cursor_to_dict,
+            query
+        )
```

Comment:
For simple selects it is probably worth trying to be consistent and use `_simple_select`. Though I think it also suffers from the problem of not correctly naming transactions.